libs/EXTERNAL/libcatch/catch.hpp

```diff
@@ -6467,7 +6467,7 @@ namespace Catch {
         static bool isSet;
         static struct sigaction oldSigActions [sizeof(signalDefs)/sizeof(SignalDefs)];
         static stack_t oldSigStack;
-        static char altStackMem[SIGSTKSZ];
+        static char altStackMem[32768];
```

Comment:
Why did you have to remove the `SIGSTKSZ` variable? This is a system constant that should be available in Ubuntu 24.04. This is also in the labcatch directory which is not maintained by VTR.

Reply:
Yes , that is what i expected too at start . it is issue with [catch1](https://github.com/catchorg/Catch2/issues/2178)  this is  solution they gave as temporary fix for catch 1 and updated the fix in catch2 gracefully 

issue is "static char altStackMem[SIGSTKSZ];" expects expression to be constant and to be in compile time 
in latest glibc  it is "# define SIGSTKSZ sysconf (_SC_SIGSTKSZ)" which loads in runtime due to sysconf syscall 

I was using master branch from the start but i happen to use v8 due to other dependancies in my usecase 

Just raised this pull request as it is broken in my system and i thought it might be helpful  .

Reply:
Yes , that is what i expected  at start too . it is issue with [catch1](https://github.com/catchorg/Catch2/issues/2178)  this is  solution they gave as temporary fix for catch 1 and updated the fix in catch2 gracefully 

issue is "static char altStackMem[SIGSTKSZ];" expects expression to be constant and to be in compile time 
in latest glibc  it is "# define SIGSTKSZ sysconf (_SC_SIGSTKSZ)" which loads in runtime due to sysconf syscall 

I was using master branch from the start but i happen to use v8 due to other dependancies in my usecase 

Just raised this pull request as it is broken in my system and i thought it might be helpful  . 
Let me know if i can be of any help with adding CI